.azure/scripts/setup-kubernetes.sh

```diff
@@ -38,6 +38,11 @@ if [ "$TEST_CLUSTER" = "minikube" ]; then
     else
         TEST_MINIKUBE_URL=https://github.com/kubernetes/minikube/releases/download/${TEST_MINIKUBE_VERSION}/minikube-linux-amd64
     fi
+
+    if [ "$KUBE_VERSION" = "latest" ]; then
+        KUBE_VERSION=$(curl -s https://github.com/kubernetes/kubernetes/releases/latest | grep -o "[0-9].[0-9][0-9].[0-9]")
+    fi
+
```

Comment:
I would prefer to do not set `--kubernetes-version` in that case instead of getting a version from a website which could be different than minikube supports (at least AFAIU from our offline conversation)

Reply:
That is quite common that the latest Kube release is not supported by Minikube yet.